frontend/src/packages/survey/libs/types/render-steps.type.ts

```diff
@@ -0,0 +1,12 @@
+import { type StepsType } from './steps.types.js';
+
+type RenderSteps = {
```

Comment:
rename and use this type in constants for steps config map
```suggestion
type StepConfiguration = {
```